worker/src/core/loaders/message-worker/govsg.class.ts

```diff
@@ -245,7 +218,10 @@ class Govsg {
         action: 'sendMessage',
       })
     } catch (error: any) {
-      if ((error as { errorCode: string }).errorCode === 'invalid_recipient') {
+      if (
+        (error as { errorCode: string }).errorCode === 'invalid_recipient' ||
```

Comment:
actually not super sure what `error as { errorCode: string }).errorCode === 'invalid_recipient'` is meant to catch and whether it's still necessary

Reply:
Yup correct

Reply:
We can actually change it to use the new type that u created also

Reply:
ok will do. I'm gonna refactor it I realize there are two identical `normalisePhonenumber` in the code base ugh

Reply:
actually `error as { errorCode: string }).errorCode === 'invalid_recipient'` will never be caught here, since the normalisation is done at the earlier `getMessage` stage.